README.md

```diff
@@ -38,6 +38,7 @@ This is an autogenerated python SDK for OpenFGA. It provides a wrapper around th
       - [Expand](#expand)
       - [List Objects](#list-objects)
       - [List Relations](#list-relations)
+      - [List Users](#list-users)
```

Comment:
I think we're maybe missing the docs content in the readme for this?

Reply:
Good catch, thanks! Fixed!